src/Aks/Aks/ChangeLog.md

```diff
@@ -32,6 +32,7 @@
 * Added parameter `-GpuInstanceProfile` for `New-AzAksCluster` and `New-AzAksNodePool`
 * Added parameter `-EnableUptimeSLA` for `New-AzAksCluster` and `Set-AzAksCluster`
 * Added parameter `-EdgeZone` for `New-AzAksCluster`
+* Added parameter `-AadProfile` for `New-AzAksCluster` and `Set-AzAksCluster`
```

Comment:
Please move this line to `## Upcoming Release` otherwise if it's empty our build tool won't detect that Aks has a new release.

Reply:
updated.